contract/src/sell-concert-tickets.proposal.js

```diff
@@ -41,9 +45,9 @@ export const makeTerms = (brand, baseUnit) => {
  * @param {*} config
  *
  * @typedef {{
- *   brand: PromiseSpaceOf<{ Ticket: Brand }>;
- *   issuer: PromiseSpaceOf<{ Ticket: Issuer }>;
- *   instance: PromiseSpaceOf<{ sellConcertTickets: Instance }>
+ *   brand: { Ticket: Brand };
+ *   issuer: { Ticket: Issuer };
+ *   instance: { sellConcertTickets: Instance }
  * }} SellTicketsSpace
```

Comment:
I need to put this back in. I removed this because TS wasn't able to pick up `PromiseSpaceOf` as its brought in via an ambient import but I later made it work via triple slash directives.